### DISCUSSION TITLE
Policy for amend to commits?

### DISCUSSION THREAD

**Discussion by frosteyes** [OE4T/meta-tegra#810] [General]
Hi

First at foremost - thanks for all the work in meta-tegra.
I have a local mirror where I have noticed that an amend to master just happened.

first version of "tegra-helper-scripts: set no_flash for BUP generation" had sha1 9be6bfac7b1ae7aceb8807e24f856ef7c4931d4e
The current version has sha1 d2210822bc262ccecbe6cc3c25a8d4ff5b1de900 and a small change in tegra210-flash-helper.sh

I understand the wish for doing amend to fix a small bug in previous commit, but I would prefer that it does not happen on pushed commits - e.g. no force push :)

Thanks again for all your contribution.

**Comment by madisongh** [comment]
Yeah, sorry about that, error on my part.
